### ISSUE TITLE
[FIXED] Doesn't work...

### ISSUE THREAD

**Issue by BlannyBlannyBlanc** [glacier-modding/io_scene_blood_money#1]
I imported a scene_main.zip from a mission and nothing happened, why?

I'm using Blender 4.2.3
Update: I was able to import the whole map, it's extremely laggy but now it works, lol 
![image](https://github.com/user-attachments/assets/be6964be-cc8f-424d-85c6-359cb521bca1)

I had to press **N** to locate the Blood Money menu since not a lot of it was straightforward unfortunately, chatgpt helped

**Comment by MwilliamsK** [comment]
Did import options work for you?

**Comment by BlannyBlannyBlanc** [comment]
> Did import options work for you?

yes, I had to press N to locate the menu since it wasn't shown by default, and yes the import options work for me on Blender 4.2 :)
